lib/middleware/auth.js

```diff
@@ -11,21 +11,29 @@ var log = require('../logger/logger.js');
 function appAuth(req, res, next){
   var apiKey = req.get('x-fh-auth-app');
   var envAccessKey = req.get('x-fh-env-access-key');
-
+  throw new Error('this is my db error');
   log.logger.debug("Authenticating App ", apiKey, envAccessKey, req.params);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this code after the "throw" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AUnreachableCode)